Dockerfile.python2.deploy

```diff
@@ -79,6 +72,11 @@ RUN pip install --no-cache-dir --exists-action=w --no-deps -r requirements/syste
     && pip install --no-cache-dir --exists-action=w --no-deps -r requirements/prod_py2.txt         \
     && pip install --no-cache-dir --exists-action=w --no-deps -e .
 
+# Install uwsgi statsd exporter to collect metrics from uwsgi when deployed
+WORKDIR /usr/lib/uwsgi/plugins
+RUN uwsgi --build-plugin https://github.com/Datadog/uwsgi-dogstatsd && \
+    rm -rf uwsgi-dogstatsd
```

Comment:
I think we need to `WORKDIR /data/olympia` after that (maybe we can even move the `WORKDIR` statement above, I'm not sure)